### PR TITLE
Updated Windows installation documentation

### PR DESCRIPTION
**Pull request by justanotherminh**
Updated instructions on how to create NuGet packages and added section on how to install and test NuGet packages.

### CODE REVIEW

File: docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md [hunk 1]
```diff
@@ -42,3 +42,13 @@ validate remote reports from enclaves.
 ```cmd
 cpack -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY
 ```
+
+## Install the NuGet package
+
+Before the `.nupkg` packages can be installed, they must be moved into a separate directory. Assuming this directory is `C:\oe_nuget\`, then the installation procedure is as followed:
```
Comment (mingweishih) [inline docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md:48]:
```suggestion
Before the `.nupkg` packages can be installed, they must be moved into a separate directory. Assuming this directory is `C:\oe_nuget\`, then the installation procedure is as follows:
```

**Review by mingweishih — APPROVED** [review]

File: docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md (hunk 1 above)
Comment (CyanDevs) [inline docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md]: Can it source current directory with `.` instead or is an absolute path necessary?

File: docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md [hunk 2]
```diff
@@ -42,3 +42,13 @@ validate remote reports from enclaves.
...
+
+```cmd
+Nuget.exe install "open-enclave" -OutputDirectory C:\oe -ExcludeVersion -Source C:\oe_nuget\
+```
+
+The package is installed at `C:\oe\open-enclave\openenclave`. The installation can then be tested by building and running samples with `CMAKE_PREFIX_PATH` set to `C:\oe\open-enclave\openenclave\lib\openenclave\cmake\`
```
Comment (CyanDevs) [inline docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md:54]: Was this with the extra `cpack` step?

Comment (justanotherminh) [reply docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md:54]: Oops somehow the `&& cpack` were removed. I added them back

File: docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md (hunk 1 above)
Comment (justanotherminh) [reply docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md]: I reworded this to imply that any directory that contains the `.nupkg` files are okay, including the current one.

Comment (CyanDevs) [reply docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md]: Great, that is more clear. Perhaps also add a prior example command that shows that the nupkg files were moved to there to be even more clear.

**Review by radhikaj — APPROVED** [review]

**Pull request merged**